salt/modules/ebuild.py

```diff
@@ -437,36 +439,59 @@ def list_pkgs(versions_as_list=False, **kwargs):
 
 def refresh_db():
     '''
-    Updates the portage tree (emerge --sync). Uses eix-sync if available.
+    Update the portage tree using the first available method from the following
+    list:
+
+    - emaint sync
+    - eix-sync
+    - emerge-webrsync
+    - emerge --sync
+
+    If the ``portage.sync_wait`` minion config is set, the portage tree will
+    not be synced within one day of the previous sync.  To enable this feature,
+    place the following config into /etc/salt/minion:
+
+    .. code-block:: yaml
+
+        portage.sync_wait: True
```

Comment:
I wonder if it would be more flexible to use a pillar variable here instead?

Reply:
Good idea.